bin/server.js

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env node
+
+/* global Config, Log */
+'use strict';
+
+const args = require('yargs')
+  .usage('Usage: $0 [args]')
+  .option('c', {
```

Comment:
Its hostname, its port, logging level, vault's hostname, vault's port. 

and this has the advantage of adding more configs if needed down the road.